<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDlg</class>
 <widget class="QDialog" name="MyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyDlg</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>531</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="pMainLayout">
    <item>
     <layout class="QFormLayout" name="pFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="pFirstLabel">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>First Number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="pFirstEdit">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="pSecondEdit">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pResultLabel">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="pResultEdit">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pSecondLabel">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Second Number</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="pButtonLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>98</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pAddButton">
        <property name="font">
         <font>
          <family>Expressway Rg</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pSubtractButton">
        <property name="font">
         <font>
          <family>Expressway Rg</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SUBTRACT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pMultiplyButton">
        <property name="font">
         <font>
          <family>Expressway Rg</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MULTIPLY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pDivideButton">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DIVIDE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pExitButton">
        <property name="font">
         <font>
          <family>Expressway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
